Dockerfile

```diff
@@ -38,6 +38,10 @@ RUN apt-get update \
 #------------------------------------------------------------------------------
 ADD rootfs /
 
+# Add *.softonic.com certificate
+RUN echo "softonic.com.crt" >> /usr/share/ca-certificates.conf \
```

Comment:
I downloaded it from https://oauth.softonic.com/public_key/development/. It's the one for *.softonic.com